index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		 <link rel="stylesheet" type ="text/css" href ="css/style.css">
+		<title>Intuit Accounts - Sign Up</title>
+	</head>
+	<body>
+<header>
+
+	<nav>
+		<div class="nav-text">
+			<p><ts style="color:black;">Already have an intuit account?</ts> <br> I forgot my User ID or Password </p>
+		</div>	
+		<div class="nav-button">
```

Comment:
`REQUIRED`

Sign in button is not properly aligned with `100%` Zoom. See below:

![Sign up button](https://res.cloudinary.com/bolaah/image/upload/v1563066685/github-microverse-project/code-reviews/005.gif)